projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-integration.php

```diff
@@ -70,8 +70,8 @@ function test_filtered_schedule_full_sync_cron_bad_schedule_sanitized() {
 		$this->assertEquals( Actions::DEFAULT_SYNC_CRON_INTERVAL_NAME, wp_get_schedule( 'jetpack_sync_full_cron' ) );
 	}
 
-	function test_starts_full_sync_on_user_authorized() {
-		do_action( 'jetpack_user_authorized', 'abcd1234' );
+	function test_starts_full_sync_on_site_registered() {
```

Comment:
lets rename this to test_starts_initial_sync_on_site_registered.   As it is a special type of Full Sync.